interface/src/app/node/node.tpl.html

```diff
@@ -7,13 +7,21 @@
   <div class="container">
     <div class="row">
       <div class="col-xs-6">
-        <high-weight-genes node-id="{{ctrl.id}}"></high-weight-genes>
+        <high-weight-genes node-id="{{ctrl.id}}" h-w-genes="ctrl.hWGenes"></high-weight-genes>
       </div>
 
       <div class="col-xs-6">
         <high-range-exp node-id="{{ctrl.id}}" top-exp="20"></high-range-exp>
       </div>
     </div>
+    <div class="row">
+      <div class="col-xs-6">
+        <enriched-genesets organism="ctrl.organism" h-w-genes="ctrl.hWGenes"></enriched-genesets>
```

Comment:
To make the tag more descriptive, maybe we should change the tag into:
```
<enriched-genesets organism="{{ctrl.organism}}" high-weight-genes="{{ctrl.highWeightGenes}}">
```
(suppose you will use "@" instead of "=" in the scope.